### ISSUE TITLE
Revisit Data Carpentry, maybe?

### ISSUE THREAD

**Issue by ctb** [aidanbudd/bosc2015#15]
Data Carpentry seems to be doing a bangup job of developing bioinformatics training resources --

https://github.com/datacarpentry/?utf8=%E2%9C%93&query=genomics

It has a different approach than Software Carpentry in that DC is doing discipline/domain specific workshops.  My outsider perspective is that they're doing some good community building around bioinformatics training resources, and I thought you might be unaware.  So, you might take another look and consider including.  No worries if not, just pointing you at those resources ;)

**Comment by aidanbudd** [comment]
Thanks for bringing that up again, Titus.

TL;DR - please add DC and SC (if you think that's too a good example for the list) to listOfOSBCs.md and send a pull request for it to be added to the repo.

We took DC, SC and GOBLET off the list, to keep the focus on communities delivering code in a open, collaboratively-built way.

Given that there are articles about [bioinformatics communities in general](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003972) and [specific open source bioinformatics communities](http://lens.elifesciences.org/07009/index.html), we thought it could be useful to focus the experiences collected here on a fairly specific kind of community (i.e. the collaborative-code-written kind described above). That might tease out ideas/experience/tips/advice at a different 'level" than that from a more general article, and with broader applicability than lessons from a single community.

Having said that, given that DC and SC (as far as I can see from checking out their github repos) write their content collaboratively using github, so I expect the parallels with code development here are strong and, as you say, community-building's a "thing" for them, so they surely have interesting insights/anecdotes from their work.

So, Titus - I encourage you, to add these via pull request to this file listOfOSBCs.md - we're keen to put this together with participation from people from different communities, maybe having extra people on the list of committers helps encourage others to join in.

**Comment by aidanbudd** [comment]
Oh, and, Titus, feedback/thoughts on features that may be missing from listOfFeaturesOfGreatOSBCs.md, please go ahead and edit and pull request (or comment in the issues)

**Comment by aidanbudd** [comment]
Closing this issue, to consolidate discussion of which communities to include/reach out to for the manuscript into one issue i.e. #38
